src/redisraft.h

```diff
@@ -476,6 +476,7 @@ typedef struct {
 
 typedef struct {
     bool asking;        /* if this command array is an asking */
+    int slot;           /* key slot for cluster mode, so doesn't have to be recalculated */
```

Comment:
might not be neccessary to add to struct depending on the consolidation we do noted above